bentoml/_internal/bento/bento.py

```diff
@@ -321,19 +324,32 @@ def creation_time(self) -> datetime:
 
     @inject
     def save(
-        self, bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store]
+        self,
+        bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store],
+        model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
     ) -> "Bento":
         self.flush_info()
 
         if not self.validate():
             logger.warning(f"Failed to create Bento for {self.tag}, not saving.")
-            raise BentoMLException("Failed to save Bento because it was invalid")
+            raise BentoMLException("Failed to save Bento because it was invalid.")
 
         with bento_store.register(self.tag) as bento_path:
             out_fs = fs.open_fs(bento_path, create=True, writeable=True)
             fs.mirror.mirror(self._fs, out_fs, copy_if_newer=False)
             self._fs.close()
             self.__fs = out_fs
+            track(
+                event_properties=BentoBuildEvent(
+                    bento_tag=self.info.tag,
```

Comment:
remove bento_tag here? it might be sensitive for some users

Reply:
ok

Reply:
can we track this in the CLI part?  this will miss all the cases where build failures. e.g. if `Bento.create` failed, it doesn't hit the `Bento#save` method at all.  It's also nice to know the duration of this CLI command run.